content/en/docs/protocol/aqua-identity-protocol.md

```diff
@@ -22,16 +22,16 @@ Self-Sovereign-Identity provided by Christopher Allen in shortened form here.[^4
 
 ### Self-Sovereign-Identity-Principles
 
-1.  **Existence.** Users must have an independent existence. 
-2.  **Control.** Users must control their identities.
-3.  **Access.** Users must have access to their own data.
-4.  **Transparency.** Systems and algorithms must be transparent.
-5.  **Persistence.** Identities must be long-lived.
-6.  **Portability.** Information and services about identity must be transportable.
-7.  **Interoperability.** Identities should be as widely usable as possible.
-8.  **Consent**. Users must agree to the use of their identity.
-9.  **Minimalization.** Disclosure of claims must be minimized.
-10. **Protection.** The rights of users must be protected.
+1.  **Existence:** Users must have an independent existence. 
```

Comment:
The "Users"'s need to be lowercased since they are no longer at the beginning at a sentence.